src/codelabs/async-await/index.md

```diff
@@ -176,23 +177,27 @@ when using `async` and `await`:
 * __The `await` keyword works only in `async` functions.__
```

Comment:
Reverse the order of these two lines? In the following part, you first talk about async and then await.